pipelines/sqre/infra/build_sciplatlab.groovy

```diff
@@ -92,6 +100,7 @@ notify.wrap {
           def wf = obj.workflow_runs[0]
           status = wf.status
           conclusion = wf.conclusion
+	  created_at = LocalDateTime.parse(wf.created_at,"2020-01-22T19:33:08Z")
```

Comment:
Is the indent here correct?  And I think the example string should not be there; the optional second argument is a formatter.

Reply:
Indent is probably wrong, but the second argument is needed, I think, in order to tell it what this string it's received from GH is gonna look like.

Reply:
As you say, it looks like it should be able to figure out for itself what format that date is.